api/src/events/customer.rs

```diff
@@ -42,7 +42,12 @@ impl CustomerEventHandler for Processor {
             auto_fuel: Some(false),
         };
 
-        let vault = self.fireblocks.client().create_vault(create_vault).await?;
+        let vault = self
+            .fireblocks
+            .client()
+            .post()
+            .create_vault(create_vault)
```

Comment:
I thought the post was internal its not very user friendly to call for the post client. Can this be an internal implementation? cleaner for the call to just call create_vault

Reply:
This API is better if you make it crud based and then the object so .create().vaault()

Reply:
we are not using any get endpoint so I think we can keep the fireblocks the same